src/services/getUser.js

```diff
@@ -0,0 +1,11 @@
+const request = path => {
+  return fetch(`https://api.github.com${path}`)
+    .then(res => Promise.all([res.ok, res.json()]))
+    .then(([ok, json]) => {
+      if(!ok) throw json;
+      return json;
+    });
+};
+
+export const getUser = username => request(`/users/:${username}`);
```

Comment:
Your fetch isn't working because of the colon you have here. Remove it and you'll be able to see in the network tab that you get a good response.